fenix/app/src/main/res/values/colors.xml

```diff
@@ -133,6 +134,8 @@
     <!-- Border -->
     <!-- Default, Divider, Dotted -->
     <color name="fx_mobile_border_color_primary">@color/photonLightGrey30</color>
+    <!-- Toolbar divider -->
+    <color name="fx_mobile_border_color_toolbar_divider">@color/photonLightGrey10</color>
```

Comment:
Could you move this after `fx_mobile_border_color_warning`? Sorry for this ask, just trying to order this to align with the figma stickersheet so that we can always easily go down the stickersheet and just make sure all the values are aligned whenever we need to update any color token here.